### PR TITLE
VAGOV-3026: Table paragraph graphql and templates

### PR DESCRIPTION
**Pull request by acabouet**
## Description
This PR adds a graphQL query and template for display of the table paragraph on detail pages.

This PR relys on:
- va.gov-cms updates in: https://github.com/department-of-veterans-affairs/va.gov-cms/pull/323
- graphql query updates in: https://github.com/department-of-veterans-affairs/vets-website/pull/10085

**Both of the above PRs should be merged in before merging this one otherwise the build will break.**

## Testing done
Tested locally with local data. 

## Screenshots
Screen Shot 2019-05-20 at 2 40 43 AM
Screen Shot 2019-05-20 at 2 40 47 AM
Screen Shot 2019-05-20 at 2 40 55 AM

## Acceptance criteria
- [x]Tables can be added inside Q&As,  inside accordions, or on their own.
- [x]Table fields can contain rich text of some kind, which can be added by any author.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/paragraphs/table.drupal.liquid [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            <thead>
+                <tr>
+                    {% for column in value %}
+                        {% if column %}
+                            <th>{{ column }}</th>
+                        {% endif %}
+                    {% endfor %}
+                </tr>
+            </thead>
+        {% elsif forloop.last == false %}
```
Comment (beeyayjay) [inline src/site/paragraphs/table.drupal.liquid]: what's this for? it looks like you're skipping the last row in the table.

**Comment by acabouet** [comment]
Yup I am.  The caption gets stored in the last key in the value object - I
skip it in the forloop and print it from it's dedicated field instead.

**Review by beeyayjay — COMMENTED** [review]
I don't know enough about feature flags to review that part, but everything else looks good to me.

File: src/site/paragraphs/table.drupal.liquid [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            "caption": "Test Table"
+        }
+    }
+{% endcomment %}
+
+
+<table class="va-table">
+
+    {% if entity.fieldTable.caption %}
+        <caption>{{ entity.fieldTable.caption }}</caption>
```
Comment (beeyayjay) [inline src/site/paragraphs/table.drupal.liquid]: May want to get clarity from @kevwalsh on this. I think he said `caption` would be visually hidden.

Comment (acabouet) [reply src/site/paragraphs/table.drupal.liquid]: I can take out - altho I noticed tables usually come paired with H3s that seem to function similarly to captions. Where do those get migrated to?

Comment (beeyayjay) [reply src/site/paragraphs/table.drupal.liquid]: At the moment, to captions. I think I got that wrong though. I'm going to confirm with Kev before taking it out. After that, nothing will be migrated to captions.

Comment (beeyayjay) [reply src/site/paragraphs/table.drupal.liquid]: Kev says you're right. I won't migrate the paired H3s into caption, but anything graphql finds in caption does belong in `<caption>`. So the above code is correct.

**Review by beeyayjay — APPROVED** [review]
LGTM!

File: src/site/paragraphs/table.drupal.liquid [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+                ],
+            }
+            "format": "rich_text",
+            "caption": "Test Table"
+        }
+    }
+{% endcomment %}
+
+
+<table class="va-table">
```
Comment (jbalboni) [inline src/site/paragraphs/table.drupal.liquid]: Do we need this `va-table` class? I don't see it in the design.va.gov table example code.

**Review by jbalboni — APPROVED** [review]

**Pull request merged**